ts/InboxRowViewModel.ts

```diff
@@ -8,6 +8,7 @@ type InboxRowViewModel = {
     imageUrl?: string
     createdAt?: Date
     read: boolean
+    last: boolean
```

Comment:
I mentioned this in another comment, but I don't think this is a good design, because this type is representing a singular row, and `last` is something that is understood in an array/collection of rows. It's also not being maintained in a consistent, intuitive way; I think it could end up causing bugs later down the line.

Reply:
Agreed. Removing from the InboxRowViewModel.